kivymd/uix/menu/menu.py

```diff
@@ -493,7 +494,6 @@ def build(self):
 ) as kv_file:
     Builder.load_string(kv_file.read())
 
-
```

Comment:
For what reason did you remove the newline?
PEP8 require classes to be separated by two indents.